redaxo/src/core/lib/response.php

```diff
@@ -324,6 +324,16 @@ public static function sendContent($content, $contentType = null, $lastModified
         }
     }
 
+    /**
+     * @param mixed  $data         data to be json_encoded() and sent
```

Comment:
```suggestion
     * @param mixed  $data         data to be json encoded and sent
```

`json_encoded` ist ja keine Funktion, sondern nur ohne "d" hinten, deswegen fand ich das so komisch.